### PR TITLE
first responsive website

### PR DESCRIPTION
**Pull request by Alexboedtker**
My biggest challenge for this project was creating the hamburger menu that transitioned into a dropdown as well as turning the hamburger icon into an X. I tried multiple variations with buttons the dropdown feature etc however they all did not work. Finally I found a great youtube video that helped me create the version I have incorporated today! 

In general, I like to focus on the features of my website because I want it to be simple and clean to navigate!

### CODE REVIEW

File: project-p2-layouts-master/code/about.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                <ul>
+                <li><a href="./index.html">HOME</a></li>
+                <li><a href="./about.html">ABOUT</a></li>
+                <li><a href="./essays.html">ESSAYS</a></li>
+                <li><a href="./letters.html">LETTERS</a></li>
+                </ul>
+              </div>
+            </div>
+          </div>
+        </div>
```
Comment (HIPPIEKICK) [inline project-p2-layouts-master/code/about.html]: Remember indentation - check out [code guidelines](https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-69ffb59eac4e4358a7c5752efb873c4e) to learn more.

```
<div class="hamburger-dropdown-menu">
  <input type="checkbox" class="toggler">
  <div class="hamburger">
    <div></div>
  </div>
  <div class="menu">
    <div>
      <div>
        <ul>
          <li>
            <a href="./index.html">HOME</a>
          </li>
          <li>
            <a href="./about.html">ABOUT</a>
          </li>
          <li>
            <a href="./essays.html">ESSAYS</a>
          </li>
          <li>
            <a href="./letters.html">LETTERS</a>
          </li>
        </ul>
      </div>
    </div>
  </div>
</div>
```

File: project-p2-layouts-master/code/about.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        <nav class="desktop-menu">
+          <a href="./index.html">HOME</a>
+          <a href="./about.html">ABOUT</a>
+          <a href="./essays.html">ESSAYS</a>
+          <a href="./letters.html">LETTERS</a>
+        </nav>
```
Comment (HIPPIEKICK) [inline project-p2-layouts-master/code/about.html:33]: Perfect indentation ⭐

File: project-p2-layouts-master/code/about.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      </header>
+    <main>
+      <div class="main-container">
+        <p>Content coming soon, stay tuned!</p>
+    </main>
```
Comment (HIPPIEKICK) [inline project-p2-layouts-master/code/about.html]: You forgot the closing tag for the `div`
```
<main>
  <div class="main-container">
    <p>Content coming soon, stay tuned!</p>
  </div>
</main>
```

File: project-p2-layouts-master/code/contact.css [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+    margin: 0;
+  }
+  
+  /*** Header ***/
+  
+  header {
+    display: flex;
+    background: #24292d;
+    height: 60px;
+  }
```
Comment (HIPPIEKICK) [inline project-p2-layouts-master/code/contact.css:14]: Indentation - see code guidelines for specification

File: project-p2-layouts-master/code/about.html [hunk 5]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <title>Responsive Website Project</title>
+    <link rel="stylesheet" href="./about.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans+Condensed:wght@300&display=swap" rel="stylesheet">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (HIPPIEKICK) [inline project-p2-layouts-master/code/about.html:8]: ⭐

**Review by HIPPIEKICK — COMMENTED** [review]
Great job implementing the hamburger! That must've taken some time - be proud! Your comments in the CSS file makes it look super structured ⭐ Keep doing that and take a look at the code guidelines to make your code look even more structured and clean.

File: project-p2-layouts-master/code/contact.css (hunk 4 above)
Comment (Alexboedtker) [reply project-p2-layouts-master/code/contact.css:14]: Hi Mathilda, Thank you so much for the feedback! It helps a lot :) I'm slightly confused as to what I need to change here. I looked at the code guidelines but don't see the issue with the indentation? 

Could you be more specific please? Thank you :)

Comment (HIPPIEKICK) [reply project-p2-layouts-master/code/contact.css:14]: Oh yes, sure! The closing tag should be on the same level of indentation as the opening line. If the code is not in media queries you don't have to indent the element.

Example 1. Here I took away the indentation from the closing tag
```
body {
  background: #E6E8E9;
  color: #000000;
  font-family: 'Open Sans Condensed', sans-serif;
  margin: 0;
}
```

Example 2. Here I took away the indentation from the whole element.
```
header {
  display: flex;
  background: #24292d;
  height: 60px;
}
```

If you were to have a media query, you should indent your element like this:

```
@media only screen and (max-width: 600px) {
  body {
    background-color: lightblue;
  }
}
```

Comment (Alexboedtker) [reply project-p2-layouts-master/code/contact.css:14]: This makes a lot more sense! I see it now :) Thank you very much!!

**Pull request closed without merging**